src/pytest_postgresql/executor.py

```diff
@@ -176,10 +176,12 @@ def wait_for_postgres(self):
         start_info = 'database system is ready to accept connections'
```

Comment:
then this won't be needed.

Reply:
Also, line 173, will we need logfile check here?

Reply:
both deleted